static.html

```diff
@@ -3369,10 +3366,25 @@ <h1 id="title" class="title">Web of Things (WoT) Discovery</h1>
         
         <p>Special thanks to Arne Broering, Soumya Kanti Datta, and Christian Bonnet
            for their survey work on existing discovery mechannisms in IoT.
```

Comment:
As already mentioned in https://github.com/w3c/wot-discovery/pull/489/files#r1204112219, there is a small typo in this line: 

```suggestion
           for their survey work on existing discovery mechanisms in IoT.
```

Reply:
Is in the report, to fix this I will have to correct it in the template.  Will do so in the next PR.